runtime/compiler/p/codegen/J9PPCInstruction.cpp

```diff
@@ -135,10 +136,20 @@ uint8_t *TR::PPCDepImmSymInstruction::generateBinaryEncoding()
             }
          }
 
-      if (cg()->comp()->compileRelocatableCode() && label == NULL)
+      if ((cg()->comp()->compileRelocatableCode() || cg()->comp()->isOutOfProcessCompilation()) &&
```

Comment:
Unfortunetaly, in master we don't have `comp()->isOutOfProcessCompilation()` by default so we have to protected it with #ifdef JITSERVER_SUPPORT

Reply:
Okay. I'll add the ifdef here and in other related PRs.